_server/requirements.txt

```diff
@@ -30,7 +30,7 @@ PyMySQL==0.10.0
 python-dateutil==2.8.1
 pytz==2020.1
 requests==2.24.0
-scikit-learn==0.23.2
+scikit-learn==0.22.2.post1
```

Comment:
why did we downgrade version to 22.2 post version here?

Reply:
ML model is in 22.2.post